### PR TITLE
HPCC-12782 Disable default for jvm using huge pages

### PR DESCRIPTION
**Pull request by mckellyln**
Signed-off-by: Mark Kelly mark.kelly@lexisnexis.com

### CODE REVIEW

**Comment by mckellyln** [comment]
@ghalliday since you reported it please review I suppose if you get a chance.

**Comment by ghalliday** [comment]
@richardkchapman the change looks sensible.  The downside would be that if thejava functions that we call use lots of memory they might be less efficient - but if that was the case they can override it.
Can you check the space change though.

File: plugins/javaembed/javaembed.cpp [hunk 1]
```diff
@@ -121,9 +121,13 @@ static class JavaGlobalState
...
         }
 
+        // Options we should set (but allow for override with jvmoptions below)
+        optionStrings.append("-XX:-UseLargePages");
+
         if (conf && conf->hasProp("jvmoptions"))
         {
-            optionStrings.appendList(conf->queryProp("jvmoptions"), ENVSEPSTR);
+            // Use space as field sep as ':' and ';' are valid
+            optionStrings.appendList(conf->queryProp("jvmoptions"), " ");
```
Comment (richardkchapman) [inline plugins/javaembed/javaembed.cpp]: This is going to break anyone's existing conf file settings, so will require a mention in the redbook at least

Comment (mckellyln) [inline plugins/javaembed/javaembed.cpp]: OK, will send info to @lorraineachapman about it.

**Comment by richardkchapman** [comment]
Merged manually into 5.2.0

**Pull request closed without merging**